[2021-03-12 10:33:48,741] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: conversion_flow.taskUploadResult 2021-03-12T10:33:19+00:00 [queued]>
[2021-03-12 10:33:48,761] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: conversion_flow.taskUploadResult 2021-03-12T10:33:19+00:00 [queued]>
[2021-03-12 10:33:48,761] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-12 10:33:48,761] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-12 10:33:48,761] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-12 10:33:48,771] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): taskUploadResult> on 2021-03-12T10:33:19+00:00
[2021-03-12 10:33:48,773] {standard_task_runner.py:52} INFO - Started process 1121 to run task
[2021-03-12 10:33:48,777] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'conversion_flow', 'taskUploadResult', '2021-03-12T10:33:19+00:00', '--job-id', '16', '--pool', 'default_pool', '--raw', '--subdir', '/srv/app/src/ckanext-mysql2mongodb/ckanext/mysql2mongodb/data_conv/dag/conversion_flow.py', '--cfg-path', '/tmp/tmpe3xqh4s2', '--error-file', '/tmp/tmpwkylivxq']
[2021-03-12 10:33:48,777] {standard_task_runner.py:77} INFO - Job 16: Subtask taskUploadResult
[2021-03-12 10:33:48,813] {logging_mixin.py:104} INFO - Running <TaskInstance: conversion_flow.taskUploadResult 2021-03-12T10:33:19+00:00 [running]> on host 345538432daf
[2021-03-12 10:33:48,840] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=conversion_flow
AIRFLOW_CTX_TASK_ID=taskUploadResult
AIRFLOW_CTX_EXECUTION_DATE=2021-03-12T10:33:19+00:00
AIRFLOW_CTX_DAG_RUN_ID=conversion_flow1615545198.950674
[2021-03-12 10:33:49,031] {python.py:118} INFO - Done. Returned value was: None
[2021-03-12 10:33:49,038] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=conversion_flow, task_id=taskUploadResult, execution_date=20210312T103319, start_date=20210312T103348, end_date=20210312T103349
[2021-03-12 10:33:49,053] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-12 10:33:49,069] {local_task_job.py:146} INFO - Task exited with return code 0
